packages/grafana-ui/src/components/Layout/Grid/GridItem.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+
+import { useStyles2 } from '../../../themes';
+
+export interface GridItemProps {
+  children: NonNullable<React.ReactNode>;
+  /** If false will the grid item inherits the display of the parent. 
+    If true, the display with 'contents' as value. See info: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_contents*/
+  displayContents?: boolean;
+  /** Specifies the column where the grid item starts within the grid */
+  columnStart?: number;
+  /** Specifies the  column where the grid item ends. 
+    If its value is span <number> the item spans across the provider number of columns*/
+  columnEnd?: number | `span ${number}`;
+  /** Specifies the row where the grid item starts within the grid */
+  rowStart?: number;
+  /** Specifies the row where the grid item ends.*/
```

Comment:
```suggestion
  /** Specifies the row where the grid item ends */
```
Consistent punctuation.